datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearch.tsx

```diff
@@ -165,12 +168,16 @@ export const EmbeddedListSearch = ({
         skip: true,
     });
 
+    const userContext = useUserContext();
+    const selectedViewUrn = userContext.localState?.selectedViewUrn;
```

Comment:
Would love to get stuff like this into custom hooks, but out of scope of this PR